prometheus-pusher/metrics.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+type MetricsPublisher struct {
+	metricsTotalPushed      prometheus.Counter
+	metricsTotalSuccess     prometheus.Counter
+	metricsTotalFailed      prometheus.Counter
+	metricsTotalSkipped     prometheus.Counter
+	metricsAnomalyGenerated *prometheus.CounterVec
```

Comment:
Metrics for push latency?

Reply:
I was thinking that but it may not useful because it is pushing to  pushgateway  and metrics will scrape based on prometheus